### ISSUE TITLE
Can't use new debug feature, worker doesn't load on localhost

### ISSUE THREAD

**Issue by c58** [rtsao/styletron#254]
I'm trying to integrate new debug feature to my storybook. I'm doing the following:
```js
import React from 'react';
import { Provider as StyletronProvider, DebugEngine } from 'styletron-react';
import { Client as StyletronClient } from 'styletron-engine-atomic';
import { configure, addDecorator } from '@storybook/react';

// Decorate with styletron
const styletron = window.styletronClient || new StyletronClient();
const debug = window.styletronDebug || new DebugEngine();
window.styletronClient = styletron;
window.styletronDebug = debug;
addDecorator(story => (
  <StyletronProvider value={styletron} debug={window.styletronDebug}>
    {story()}
  </StyletronProvider>
));
```

But i'm getting the following error in the console:
```
Uncaught DOMException: Failed to construct 'Worker': Script at 'https://unpkg.com/css-to-js-sourcemap-worker@2.0.1/worker.js' cannot be accessed from origin 'http://localhost:6006'.
    at new DebugEngine (http://localhost:6006/static/preview.bundle.js:54778:16)
    at Object.<anonymous> (http://localhost:6006/static/preview.bundle.js:76317:38)
    at Object.<anonymous> (http://localhost:6006/static/preview.bundle.js:76336:30)
    at __webpack_require__ (http://localhost:6006/static/preview.bundle.js:708:30)
    at fn (http://localhost:6006/static/preview.bundle.js:113:20)
    at Object.window.STORYBOOK_REACT_CLASSES (http://localhost:6006/static/preview.bundle.js:75004:18)
    at __webpack_require__ (http://localhost:6006/static/preview.bundle.js:708:30)
    at http://localhost:6006/static/preview.bundle.js:806:40
    at http://localhost:6006/static/preview.bundle.js:809:10
```

Any clues?

**Comment by rtsao** [comment]
Thanks for pointing this out, I've created PR to address this: https://github.com/rtsao/styletron/pull/255

I think this should work for many use cases, but CSP is always tricky. For the record, the way I'm actually using this feature is by having [the web app actual serve the webworker so it is same-origin](https://github.com/fusionjs/fusion-plugin-styletron-react/pull/122/files).

**Comment by rtsao** [comment]
@c58 The fix in #255 has been released in `styletron-react-core@1.3.1`/`styletron-react@4.3.1`. I think this should resolve your problem.

For most SPA tools (e.g. storybook, create-react-app, etc.) this will work. For apps that set CSP headers, a different approach for loading the web worker (and wasm source) may be needed.
